doc/sphinx/install/ubuntu.md

```diff
@@ -1,13 +1,12 @@
 (sec-install-ubuntu)=
 # Ubuntu Packages
 
-As of Cantera 3.1.0, packages are available for Ubuntu 20.04 (Focal Fossa), Ubuntu 22.04
-(Jammy Jellyfish), Ubuntu 24.04 (Noble Numbat) and Ubuntu 24.10 (Oracular Oriole).
-Generally, packages are available for the two most-recent LTS releases as well as the
-non-LTS releases supported on [Launchpad](https://launchpad.net/ubuntu) at the time of
-the Cantera release. To see which Ubuntu releases and Cantera versions are currently
-supported, visit the
-[Cantera PPA](https://launchpad.net/~cantera-team/+archive/ubuntu/cantera).
+As of Cantera 3.2.0, packages are available for Ubuntu 24.04 (Noble Numbat) and Ubuntu
```

Comment:
Just for clarification, we're not going to support 22.04, which would be "the last two LTS releases", but we will support 26.04 next spring to restore the last two LTS goal?

Reply:
I revised this to limit expectations to one LTS release and one non-LTS release.